packages/site/src/ui/doc.js

```diff
@@ -1,9 +1,10 @@
-import * as core from '@pluralsight/ps-design-system-core'
 import Markdown from 'react-markdown/with-html.js'
 import PropTypes from 'prop-types'
 import React from 'react'
 import Router from 'next/router.js'
 
+import * as core from '@pluralsight/ps-design-system-core'
```

Comment:
this used?

Reply:
yes. but I didn't end up changing this file so i shouldn't have commited this file